mods/ts/rules/nod-support.yaml

```diff
@@ -16,6 +16,8 @@ NAWALL:
 		CrushClasses: heavywall
 	LineBuild:
 		NodeTypes: wall, turret
+	Replaceable:
+		Type: nawall
```

Comment:
Didn't you also want to support placing walls on top of damaged walls to replace them?

Reply:
I just tried it and discovered a funny bug. When replacing a wall, it gets removed, then build and build a second time ontop (couse of linebuild, i assume).